cli/cmd/plugin/ui/web/tanzu-ui/src/shared/types/types.ts

```diff
@@ -1,5 +1,7 @@
 import { Dispatch, Reducer, ReducerAction } from 'react';
-
+import { RESOURCE } from '../../state-management/actions/Resources.actions';
```

Comment:
please sort all imports in alphabetical order.

Reply:
Yes sir